pkg/model/sdk_helper.go

```diff
@@ -382,20 +386,26 @@ func (a *SDKAPI) SDKAPIInterfaceTypeName() string {
 }
 
 // Override the operation type and/or resource name if specified in config
-func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) (OpType, string) {
+func getOpTypeAndResourceName(opID string, cfg *ackgenconfig.Config) ([]OpType, string) {
+	var unmarshaledOpTypes ackgenconfig.StringArray
 	opType, resName := GetOpTypeAndResourceNameFromOpID(opID)
+	opTypes := []OpType{opType}
 
-	if cfg != nil {
-		if operationConfig, exists := cfg.Operations[opID]; exists {
-			if operationConfig.OperationType != "" {
-				opType = OpTypeFromString(operationConfig.OperationType)
-			}
+	if cfg == nil {
+		return opTypes, resName
+	}
+	if operationConfig, exists := cfg.Operations[opID]; exists {
+		if operationConfig.ResourceName != "" {
+			resName = operationConfig.ResourceName
+		}
 
-			if operationConfig.ResourceName != "" {
-				resName = operationConfig.ResourceName
+		if b, err := json.Marshal(operationConfig.OperationType); err == nil {
+			unmarshaledOpTypes.UnmarshalJSON(b)
```

Comment:
You don't have to call the method `UnmarshalJSON` here, bur rather call `json.Unmarshal(b, &unmarshaledOpTypes)`